components/script/dom/htmllinkelement.rs

```diff
@@ -113,6 +113,9 @@ impl HTMLLinkElement {
             doc.remove_stylesheet(self.upcast(), s)
         }
         *self.stylesheet.borrow_mut() = Some(s.clone());
+        if let Some(cssom_stylesheet) = self.get_cssom_stylesheet() {
```

Comment:
All these should use `self.cssom_stylesheet.get()`, `get_cssom_stylesheet` will unnecessarily create a stylesheet.